### DISCUSSION TITLE
zeekpolicysync, Unable to manage file: Jinja syntax error

### DISCUSSION THREAD

**Discussion by lrhenness** [Security-Onion-Solutions/securityonion#7942] [Unsupported Versions] [answered]
This has occurred on both the regular ISO sensor install, as well as the so-network-setup script on top of Ubuntu.

I am installing an SO2 distributed sensor and the zeekpolicysync task fails with the following error:
```
Unable to manage file: Jinja syntax error: Encountered unknown tag 's'.; line 46`
Unable to manage file: Jinja variable `sensorname` is undefined`
```
Obviously, it looks like a syntax error but I am confused on how to resolve it. Some notes:

1. This stops the entire installation and leaves me dead in the water. 
2. This looks like it's trying to run commented code..?
3. Even if I were able to resolve this by removing the commented code, I'm not sure if the install script would just pull fresh from the repository during runtime.
4. The error says that sensorname is undefined in relation to the base (not modified) `bfpconf.zeek`. If related, I **am** setting the sensor name in the install script. The format is along the lines of `example-sensor-name` with dashes/hyphens, same as the manager which was installed previously with an older version of the SO2 ISO.

![error](https://user-images.githubusercontent.com/48068359/168298177-1b3f22ce-610f-49b9-8aca-ff989464115b.png)

My question: Is this worthy of [a new issue](https://github.com/Security-Onion-Solutions/securityonion/issues/new)? Or is there a path towards resolving on my end?

Thanks all.

**Comment by dougburks** [comment, marked as answer]
> My question: Is this worthy of [a new issue](https://github.com/Security-Onion-Solutions/securityonion/issues/new)?

I don't think this needs a new issue as we do lots of testing (both automated and manual) and don't experience this issue.

> If related, I am setting the sensor name in the install script.

What exactly do you mean here?

**Reply by lrhenness** [reply]
> I don't think this needs a new issue as we do lots of testing (both automated and manual) and don't experience this issue.

Roger.

> What exactly do you mean here?

The error states that the Jinja variable 'sensorname' is undefined. To me, this seems related to the sensor name. I can see how this might seem like I did not set the sensor name. I set the sensor name. 

Thanks for the help, Doug. I'll figure it out. Maybe it's just my hardware.

Have a good one.
